documentation/assemblies/upgrading/assembly-downgrade.adoc

```diff
@@ -0,0 +1,26 @@
+// This assembly is included in the following assemblies:
+//
+// deploying/master.adoc
+
+[id='assembly-downgrade-{context}']
+= Downgrading Strimzi
+
+If you are encountering issues with the latest version of Strimzi,
+you can revert your installation to the previous version.
+
+As with the Strimzi upgrade process, you can perform your downgrade in two stages:
+
+. Revert your Cluster Operator to the previous Strimzi version.
+** xref:proc-downgrade-cluster-operator-{context}[]
+
+. Downgrade all Kafka brokers and client applications to the previous Kafka version.
```

Comment:
This is not correct. The operator currently takes charge of the interbroker protocol, and updates it automatically. That means that the user cannot safely downgrade the brokers, even if they didn't complete the upgrade. I.e. once they've started to upgrade the Kafka cluster then they can't currently safely back out. They can upgrade and downgrade the operator.

Reply:
@tombentley Thanks Tom. I thought we could perform the downgrade if interbroker versions were the same. So you recommend we drop the section on downgrading Kafka entirely for now?

Reply:
@PaulRMellor yeah, it would work like that if the operator didn't bump the interbroker version for you. IOW once you've changed `Kafka.spec.kafka.version` you've essentially changed the interbroker version, and Kafka downgrade is no longer possible. So, yes we should drop the section on downgrading Kafka for the time being.